PrebidMobileTests/AdUnitTests/AdUnitTests.swift

```diff
@@ -36,7 +34,7 @@ class AdUnitTests: XCTestCase {
         var resultCode: ResultCode?
         
         let expected = ResultCode.prebidDemandFetchSuccess
-        let adUnit = AdUnit(configId: "138c4d03-0efb-4498-9dc6-cb5a9acb2ea4", size: CGSize(width: 300, height: 250))
```

Comment:
Need to be sure that we still have the original construction method. I saw that it was removed above.